test/variable-subsetter/VariableSubsetter_Regression.ipynb

```diff
@@ -155,97 +163,61 @@
    "outputs": [],
    "source": [
     "if var_subsetter_info is not None:\n",
-    "    no_int64_file_name = 'var_subsetter_no_int64.nc4'\n",
-    "    no_int64_request = Request(collection=var_subsetter_info['collection'],\n",
-    "                               granule_id=[var_subsetter_info['granule_id']],\n",
-    "                               variables=['/gt1l/land_segments/dem_h'])\n",
+    "    single_var_file_name = 'var_subsetter.nc4'\n",
+    "    single_var_request = Request(collection=var_subsetter_info['collection'],\n",
+    "                                 granule_id=[var_subsetter_info['granule_id']],\n",
+    "                                 variables=['<enter variable full name>'])\n",
     "\n",
-    "    submit_and_download(harmony_client, no_int64_request, no_int64_file_name)\n",
-    "    assert exists(no_int64_file_name), 'Unsuccessful non-Int64 Variable Subsetter request.'\n",
+    "    submit_and_download(harmony_client, single_var_request, single_var_file_name)\n",
+    "    assert exists(single_var_file_name), 'Unsuccessful single-variable Variable Subsetter request.'\n",
     "\n",
-    "    compare_results_to_reference_file(no_int64_file_name,\n",
-    "                              'reference_files/var_subsetter_no_int64_reference.nc4')\n",
+    "    compare_results_to_reference_file(\n",
+    "        single_var_file_name,\n",
+    "        'reference_files/var_subsetter_single_var_reference.nc4'\n",
+    "    )\n",
     "\n",
-    "    print_success('Variable subsetter synchronous request.')\n",
+    "    print_success('Variable subsetter single variable request.')\n",
     "else:\n",
     "    print(f'The Variable Subsetter is not configured for environment: \"{harmony_environment}\" - skipping test.')"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "40e40229-3ff7-4d82-945b-af33703eb11c",
+   "id": "aaeb21d3-ce95-4068-b6aa-28b9f9b93b5a",
    "metadata": {},
    "source": [
-    "### Variable Subsetter request and Int64 variables\n",
```

Comment:
The significance of testing Int64-type variables has probably long passed by now. For reference, these variables were something that DAP2 couldn't handle in OPeNDAP, but DAP4 can.